source/_drafts/Video-js-7-is-here.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Video.js 7 is here! 🎉
+author:
+  name: Gary Katsevman
+  github: gkatsev
+tags:
+---
+![](vjs 7 small.png)
+
+I'm happy to announce that Video.js 7 is now out! It brings to Video.js support for HLS and experimental DASH support via the [Video.js HTTP Streaming (VHS)][VHS] project. Video.js 7 also drops support for some older Internet Explorer browsers only keeping support for IE11.
+
+While it's out, it's still in pre-release for a short while and only available under the `next` tag on npm. Some time next week, we'll flip the switch and make it latest.
+
+![changes for removing IE8 code](changes.png)
+This release is pretty exciting because it means we were able to [remove a bunch of code](https://github.com/videojs/video.js/pull/5041) for old browsers and we can now start using newer web platform features that we had previously been avoiding like the `:not()` css selector. It'll also free us to look forward at new features and not be held back by needing to support old browsers like IE8.
+
+## Getting Video.js 7
+
+You can get it from npm via
+```sh
+npm install video.js@next
+```
+or
+```sh
+npm install video.js@7.0.0
+```
+
+From [GitHub Releases](https://github.com/videojs/video.js/releases/tag/v7.0.0),
+
+Or from our or another CDN
+```
+https://vjs.zencdn.net/7.0.0/video.min.js
+https://vjs.zencdn.net/7.0.0/video-js.css
+```
+```
+https://unpkg.com/video.js@7.0.0/dist/video.min.js
+https://unpkg.com/video.js@7.0.0/dist/video-js.css
+```
+```
+https://cdnjs.cloudflare.com/ajax/libs/video.js/7.0.0/video.min.js
+https://cdnjs.cloudflare.com/ajax/libs/video.js/7.0.0/video-js.css
+```
+
+## VHS and Video.js
+Video.js will be bundling VHS by default for ease of use for new users. However, some people don't want VHS or are using another plugin. For this, we have a separate build of Video.js which doesn't include VHS.
+You can get it from our CDN or another CDN
```

Comment:
Minor detail, but paragraphs really ought to have two newlines between them, so they result in actual `<p>`s instead of `<br>`s. Helps with readability.